src/constants/properties.constant.ts

```diff
@@ -159,6 +159,63 @@ export const PROPERTIES = {
         compBalPctReserve: 0,
         compBalPctAmount: 0,
       },
+      POST: {
+        isFinalAvailableDateMaximum: true,
+        isExpirationDateMaximum: true,
+        bookingClassCode: 'A',
+        facilityInitialStatusCode: 'P',
+        lineOfficerIdentifier: 'DCIS',
+        generalLedgerUnitIdentifier: 'ECGD',
+        servicingUnitIdentifier: 'ACBS',
+        servicingUnitSectionIdentifier: 'ACBS',
+        purposeTypeCode: '   ',
+        capitalClassCode: 'A',
+        capitalConversionFactorCode: {
+          '250': '1',
+          '260': '5',
+        },
+        capitalConversionFactorCodeFallback: '1',
+        financialFXRate: 1,
+        financialFXRateOperand: 'D',
+        financialRateFXRateGroup: 'UKRATEGRP',
+        financialFrequencyCode: 'M',
+        financialBusinessDayAdjustment: 'S',
+        financialDueMonthEndIndicator: false,
+        calendarIdentifier: 'UK',
+        financialLockMTMRateIndicator: true,
+        financialNextValuationDate: '2060-12-25T00:00:00Z',
+        customerFXRateGroup: 'UKRATEGRP',
+        customerFrequencyCode: 'M',
+        customerBusinessDayAdjustment: 'S',
+        customerDueMonthEndIndicator: false,
+        customerLockMTMRateIndicator: true,
+        customerNextValuationDate: '2060-12-25T00:00:00Z',
+        limitRevolvingIndicator: true,
+        standardReferenceType: '' as const,
+        servicingUser: { userAcbsIdentifier: 'OPERATIONS', userName: 'OPERATIONS' },
+        administrativeUser: { userAcbsIdentifier: 'OPERATIONS', userName: 'OPERATIONS' },
+        isNextReviewDateZero: true,
+        isCreditReviewRiskDateZero: true,
+        regulatorRiskDate: '',
+        isRegulatorRiskDateZero: true,
+        multiCurrencyArrangementIndicator: true,
+        isUserDefinedDate3Zero: true,
+        isUserDefinedDate4Zero: true,
+        defaultReasonCode: ' ',
+        lenderTypeCode: '100',
+        nextReviewDate: '2050-01-01T00:00:00Z',
+        isOfficerRiskDateZero: true,
+        facilityUserDefinedList3Code: '2',
+        probabilityofDefault: 0,
+        doubtfulPercent: 0,
+        drawUnderTemplateIndicator: false,
+        facilityOriginationCode: 'C',
+        accountStructureCode: 'B',
+        facilityStatusCode: 'D',
+        compBalPctReserveUnissued: 75,
+        compBalPctReserveIssued: 100,
+        riskMitigationCode: '',
```

Comment:
Do we need default value for agentBankIdentifier, maybe it is not required, maybe we can use global ACBS_UKEF_PARTY_ID=00000000

Reply:
Looks like agentBankIdentifier is required field, so we don't need default value for it.